10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,103 @@
+               Heirloom meditation fugiat organic thundercats. IPhone intelligentsia thundercats, fanny pack aute poutine laboris ethical.
+            </p>
+          </div>
+          <div class="service two">
+              <img src="img/picture3.jpg" alt="BLack and white photo" />
+              <h4>Black and White Photography</h4>
+              <p>
+                 Heirloom meditation fugiat organic thundercats. IPhone intelligentsia thundercats, fanny pack aute poutine laboris ethical.
+              </p>
+          </div>
+          <div class="service three">
+              <img src="img/picture4.jpg" alt="scenic photo" />
+              <h4>Landscape Photography</h4>
+              <p>
+                 Heirloom meditation fugiat organic thundercats. IPhone intelligentsia thundercats, fanny pack aute poutine laboris ethical.
+              </p>
+          </div>
+          <div class="service four">
+              <img src="img/Boston City Flow.jpg" alt="City light photo" />
+              <h4>City lights Photography</h4>
+              <p>
+                 Heirloom meditation fugiat organic thundercats. IPhone intelligentsia thundercats, fanny pack aute poutine laboris ethical. 
+              </p>
+          </div>
+        </section>
+        <section id="Testimonials" class="clearfix">
+            <div class="test uno">
+                <img src="img/DSC_0090_3.jpg" alt="Vacation picture" />
+                <p>
+                    Thank you for capturing every moment of our vacation, cant get enough of this picture! Its too Damn funny!!- Alex
+                </p>
+            <h6>04/13/2014</h6>
+            </div>
+            <div class="test dos">
+                <img src="img/pi5.JPG" alt="CUte kids" />
+                <p>
+                    This picture speaks for itself, loved the you click!A precious moment captured which will touch my heart forever.  - Ritu
+                </p>
+                <h6>08/06/2015</h6>
+            </div>
+        </section>
+        <section id="Contactme">
+            <h2>If you need a picture taken for a party, for headshots and if you just need a family vacation covered.<br>MaIl Me! </h2>
+            <a href="#Email">akshayasaraf@gmail.com</a>
+            <br>
+            <div class="socialmedia">
+            <a href="#facebook"><img src="https://cdn1.iconfinder.com/data/icons/logotypes/32/square-facebook-128.png" alt="facebook"/></a>
+            <a href="#instagram"><img src="http://3835642c2693476aa717-d4b78efce91b9730bcca725cf9bb0b37.r51.cf1.rackcdn.com/Instagram_App_Large_May2016_200.png" alt="instagram" /></a>
+            <a href="#twitter"><img src="http://icons.iconarchive.com/icons/limav/flat-gradient-social/512/Twitter-icon.png" alt="twitter" /></a>
+            <a href="#pintrest"><img src="https://cdn0.iconfinder.com/data/icons/Pinterest/512/Pinterest_Favicon.png" alt="pinterest" /></a>
+            </div>
+        </section>
+    </main>
 </body>
+
 </html>
```

Comment:
Wonderful use of the assignment - and hopefully you got a head start on your portfolio site. Well done.